src/lib/ClaimCodeManager.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import crypto from 'crypto';
+
+type ClaimCodeObject = {
+  claimCode: string,
+  userID: string,
+}
+
+const CLAIM_CODE_LENGTH = 63;
+const CLAIM_CODE_LIVE_TIME = 5000 * 60; // 5 min
```

Comment:
@AntonPuko i like it, i think that is the right way. About TTL i'm going to make some test cases.